Source/UnitTests/UnitTests.vcxproj.user

```diff
@@ -5,5 +5,6 @@
     <!--This is only used to allow UnitTests to find OpenAL DLL...kinda hacky-->
     <LocalDebuggerWorkingDirectory>$(BinaryOutputDir)</LocalDebuggerWorkingDirectory>
     <DebuggerFlavor>WindowsLocalDebugger</DebuggerFlavor>
+    <ShowAllFiles>true</ShowAllFiles>
```

Comment:
Please remove this from the commit; it seems unlikely that we want this (otherwise it would've been here already)